mozci/sources/pushlog.py

```
@@ -134,12 +139,32 @@ def query_revision_info(repo_url, revision, full=False):
     return push_info
 
 
+def query_full_revision_info(repo_name, revision):
+    """
+    Return the full 40 chars revision number which correspond to the 12 chars
+    revision number we given, and throw an error out if found collision in there.
+
+    :param repo_name: The name of a repository e.g. mozilla-inbound
+    :type repo_name: str
+    :param revision: push revision (40 chars)
+    :type revision: str
+    :return:
+    """
+    if len(str(revision)) <= 12:
+        url = query_repo_url(repo_name)
+        try:
+            revision = query_revision_info(url, revision)['changesets'][0]
+        except AssertionError:
+            print "please check the revision number you had entered."
```

Could you please raise as PushlogError instead of a print?